### PR TITLE
initial fixture data to test use-cases

### PR DESCRIPTION
**Pull request by teslajoy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -169,3 +169,5 @@ clinical_reports/
 output
 studies/
 .python-version
+
+tests/fixtures/.DS_Store
```
Comment (quinnwai) [inline .gitignore:173]: maybe change to `.DS_Store` so it ignore a .DS_Store in any directory?

File: tests/fixtures/fhir-compbio-examples/META/Observation.ndjson [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+{"resourceType":"Observation","id":"21f3411d-89a4-4bcc-9ce7-b76edb1c745f","status":"final","category":[{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/observation-category","code":"laboratory","display":"Laboratory"}]}],"code":{"coding":[{"system":"https://loinc.org","code":"81247-9","display":"Genomic structural variant copy number"}]},"subject":{"reference":"Patient/bc4e1aa6-cb52-40e9-8f20-594d9c84f920"},"focus":[{"reference":"DocumentReference/9ae7e542-767f-4b03-a854-7ceed17152cb"}],"specimen":{"reference":"Specimen/60c67a06-ea2d-4d24-9249-418dc77a16a9"},"component":[{"code":{"coding":[{"system":"https://my_demo.org/labA","code":"Gene","display":"Gene"}],"text":"Gene"},"valueString":"TP53"},{"code":{"coding":[{"system":"https://my_demo.org/labA","code":"Chromosome","display":"Chromosome"}],"text":"Chromosome"},"valueString":"chr17"},{"code":{"coding":[{"system":"https://my_demo.org/labA","code":"result","display":"result"}],"text":"result"},"valueString":"gain of function (GOF)"}]}
```
Comment (quinnwai) [inline tests/fixtures/fhir-compbio-examples/META/Observation.ndjson:3]: Ah thanks! Cool to know that multiple observations can map to a single document reference, this helps clarify the "asssociate files with observation/specimen" use case.

File: tests/fixtures/fhir-compbio-examples/META/ResearchStudy.ndjson [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "identifier": [
+    {
+      "use": "official",
+      "system": "https://my_demo.org/labA",
+      "value": "labA"
+    }
+  ],
+  "name": "LabA",
+  "status": "active",
+  "description": "LabA Clinical Trial Study: FHIR Schema Chorot Integration"
```
Comment (quinnwai) [inline tests/fixtures/fhir-compbio-examples/META/ResearchStudy.ndjson:13]: intentional or "Cohort" typo?

File: tests/fixtures/fhir-compbio-examples/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+```
+>>>> resources={'summary': {'DocumentReference': 1, 'Specimen': 1, 'Observation': 3, 'ResearchStudy': 1, 'ResearchSubject': 1, 'Organization': 1, 'Patient': 1}}
+```
+
+There are three Observations with user-defined metadata component. 
+1. Focus - reference -> Specimen
+2. Focus - reference -> DocumentReference 
+   1. The first Observation contains metadata on the file's sequencing metadata.
+   2. The second Observation includes a simple summary of a CNV analysis result computed from this file.
```
Comment (quinnwai) [inline tests/fixtures/fhir-compbio-examples/README.md:11]: thanks for the docs! Very helpful

File: tests/fixtures/fhir-compbio-examples/META/Specimen.ndjson [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            "code": "117032008",
+            "display": "Spun specimen (procedure)"
+          },
+          {
+            "system": "https://my_demo.org/labA",
+            "code": "Double-Spun",
+            "display": "Double-Spun"
+          }
+        ],
+        "text": "Spun specimen (procedure)"
```
Comment (quinnwai) [inline tests/fixtures/fhir-compbio-examples/META/Specimen.ndjson:46]: Do we have standardization on how we are using system/code/display/text/value for CodeableConcepts btw?

**Review by quinnwai — APPROVED** [review]
minor comment, rest for my understanding, lgtm!

**Comment by bwalsh** [comment]
merging.

**Pull request merged**